datasets/streaming/error_aggregates/reference.md

```diff
@@ -0,0 +1,93 @@
+# Error Aggregates Reference
+
+<!-- toc -->
+
+# Introduction
+
+{% include "./intro.md" %}
+
+# Data Reference
+
+## Sampling
+
+### Invalid Pings
+
+We ignore invalid pings in our processing. Invalid pings are defined as those that:
+
+* The submission dates or activity dates are invalid or missing.
+* The build ID is malformed.
+* The docType field is missing or unknown.
+* The ping does not include a positive count of usage hours.
```

Comment:
@fbertsch could use some confirmation on this, I cribbed this section from crash_aggregates. I can easily see that the 3rd and 4th clauses are true from the source. The first and second are assumptions based on how we validate pings via the schema?

Reply:
@wlach I'm not sure it's worth it to enumerate the various reasons we discard pings, the schema is pretty complex (and does include e.g. missing submission_date, malformed builds, etc.).

For this dataset in particular, we only process `main` and `crash` pings. Number 4 is true.

Reply:
Yeah, I think you're right. It's probably just a background assumption that we discard this kind of invalid data, probably not really relevant to the intended audience here. I'll remove it.